VARIABLES.md

```diff
@@ -838,15 +878,15 @@ Default:  "{{ monitoring_interceptors_enabled }}"
 
 ### ksql_user
 
-Only use to customize Linux User ksqlDB Service runs with. User must exist on host.
+Only use to customize Linux User ksqlDB Service runs with. Default user is cp-ksql
 
 Default:  "{{ksql_default_user}}"
 
 ***
 
 ### ksql_group
 
-Only use to customize Linux Group ksqlDB Service user belongs to. Group must exist on host.
+Only use to customize Linux Group ksqlDB Service user belongs to. Default group is confluent
```

Comment:
Modify this variable to customize the Linux Group that the ksqlDB Service user belongs to. Default group is confluent.